python/Quickstart.MD

```diff
@@ -5,130 +5,107 @@ This quickstart covers:
 1. Creating a direct supplier incident using the APT-SCWM API
 2. Using the response data to add a comment to the created incident   
 
-## Request Body
+*This example uses Python version 3.10 or later entering commands in a Python terminal.*  
 
-For this example, we are working backwards from the request body. In its simplest form, the request body contains a header and a payload.  A simple request body resembles the following:
-```
-{
-    "header": {
-        "headerVersion": 1,
-        "eventName": "agile-process-teams:add-direct-supplier-incident:v2",
-        "ownerId": "94f94f37-2772-4b39-8041-9c2dcfcfff82",
-        "processNetworkId": "945feb8e-09a7-4fef-9a8a-b3c5b56f87d",
-        "appName": "agile-process-teams",
-        "dataspace": "default"
-    },
-    "payload": {
-        "aptBusinessObjectSummary": "Raw materials shortage at Location XYZ",
-        "directSupplierImpact": {
-            "businessPriority": "MEDIUM"
-        }
-    }
-}
-```
+**To follow along with the example, open your preferred Python terminal and use a single session through the entire process.**  
 
-Many requests share common values in the header. The `headerVersion`, `appName`, and `dataspace` do not change. For a single company, the `ownerId` and `processNetworkId` values are likely to remain the same. This means you may only need to determine the `eventName`. The payload will change with each new request depending on the selected `eventName`. As the integrator, you can store these values after creation or generate and use them dynamically at runtime.  
+## Create Your Request  
 
-### Create Payload File
 
-The APT-SCWM API uses JSON for passing data. The minimum required data to submit a direct supplier incident is a summary of the incident and an impact priority level.  An example of the minimum required payload:
+### Request Body  
+
+For this example, we create the request body first.  In its simplest form, the request body contains a header and a payload and conforms to this format:
 ```
 {
-"payload": {
-    "aptBusinessObjectSummary": "Automation test from samples folder.",
-     "directSupplierImpact": {
-        "businessPriority": "LOW"
-        }
-    }
+  "header":{},
+  "payload":{}
 }
 ```
-
-However, for this example, we will include more than the miniumum information. Create a payload file containing the below and name it `addIncident.json`.  
+**Type or copy the following into your Python terminal**  
```

Comment:
**Type or copy the following into your Python terminal, replacing necessary values**

Reply:
Updated as suggested.